app/assets/javascripts/donations.js

```diff
@@ -0,0 +1,30 @@
+var main_height = $('main').height();
+
+var scrolling_support = false;
+function checkScrollingSupport() {
+  if($(window).width() > 600) {
+    $("main").addClass("support-scrolling");
+    scrolling_support = true;
+  }
+}
+
+// Scroll the donation box so it's always in view
+function scrollHandler(event) {
+  checkScrollingSupport();
+  var h = $("main").height();
+  var s_top = $(window).scrollTop();
+  var d_box = $(".donations");
+  var bounded_scroll = Math.max(55, Math.min(s_top, h - d_box.outerHeight(true)));
```

Comment:
Line is too long.<br>Identifier 'bounded_scroll' is not in camel case.<br>Identifier 's_top' is not in camel case.<br>Identifier 'd_box' is not in camel case.